draft-ietf-rtcweb-jsep.xml

```diff
@@ -4444,6 +4447,152 @@ a=msid:-
         </t>
       </section>
     </section>
+    <section title="Early Transport Warmup Example" anchor="sec.warmup-example" >
+
+      <t>This example demonstrates the early warmup technique described in
+      <xref target="sec.use-of-provisional-answer"/>. Here, Alice's endpoint
+      sends an offer to Bob's endpoint to start an audio/video call. Bob
+      immediately responds with an answer that accepts the audio/video
+      m= sections, but marks them as sendonly (from its perspective), meaning
+      that Alice will not yet send media. This allows the browser to start
+      negotiating ICE and DTLS immediately. Bob's endpoint then prompts him
+      to answer the call, and when he does, his endpoint sends a second offer
```

Comment:
I think endpoint is probably most appropriate here - only applications/endpoints can trigger the sending of an offer.